changebot/changelog.py

```diff
@@ -25,8 +25,12 @@ def find_prs_in_changelog_by_section(content):
     subcontent = ''
     previous = None
 
+    new_changelog_format = False
+    
     for line in content.splitlines():
-        if '-------' in line:
+        if '=======' in line:
+            new_changelog_format = True
+        if '=======' in line or (not new_changelog_format and '-------' in line):
```

Comment:
Well... It looks slightly less repetitive if you replace `'=======' in line` here with `new_changelog_format`. I don't see any easy solution to get rid of the double-if without refactoring the rest below (and even then might still look ugly).